### PR TITLE
Labeeba first

### PR DESCRIPTION
**Pull request by slk18bg**
Question object shaped as requested; there are other issues though

### CODE REVIEW

File: src/pages/api/buildRecList.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { useBuildRecListService } from "./services/buildRecList.service";
+
+export default async function buildRecs(req, res) {
+  const responseId = req.body;
+  console.log({ responseId });
+  //const { recList } = useBuildRecListService({ responseId });
+  const { buildGameRecList } = useBuildRecListService( {responseId} );
```
Comment (dotGrimmz) [inline src/pages/api/buildRecList.js]: image

This should should give you a hint on why its showing up undefined in your service file.

File: src/pages/api/services/fetchData.service.js [hunk 2]
```diff
@@ -65,167 +61,80 @@ export const useFetchDataService = () => {
...
+        } else if (obj.name.includes("PlayStation")) {
+          sony.push(obj.id);
+        } else if (obj.name.includes("PC")) {
+          pc.push(obj.id);
+        } else if (obj.name.includes("Xbox")) {
+          microsoft.push(obj.id);
+        }
+      });
+
+      let platformObjs = [
```
Comment (dotGrimmz) [inline src/pages/api/services/fetchData.service.js]: Set the definition before performing methods on it for readability purposes.

**Review by dotGrimmz — CHANGES_REQUESTED** [review]
- Move the contents of your .env.local file which is in the src directory to a file called .env at the root of the application directory. That should allow the process.env.VAR_NAME to work in your api. 

  - The questionObj file is really just a template the backend 
  needs to fill out and return to the 
  front end with the correct id mappings, so feel free to import that file into your service and use it to just map over the corresponding id's.

 - The default props that should already be filled are the 
  question, name, options.name, imgSrc, and options.imgSrc.

File: src/pages/api/buildRecList.js (hunk 1 above)
Comment (slk18bg) [reply src/pages/api/buildRecList.js]: This one has been resolved! Didn't know the param names had to be the same

**Review by dotGrimmz — APPROVED** [review]

**Pull request merged**